examples/integration_test.py

```diff
@@ -56,6 +56,22 @@ def error_cb (err):
     print('Error: %s' % err)
 
 
+class SchemaRegistry(object):
```

Comment:
At first I was thinking a global ``schema_registry = SchemaRegistry()`` or ``schema_registry = CustomSchemaRegistry()`` but went with class methods. The latter looks good to me. What do you think?